### DISCUSSION TITLE
oifsaufhsanmd

### DISCUSSION THREAD

**Discussion by Alias1XP** [ScratchAddons/ScratchAddons#5805] [Ideas] [answered]
im smartest man of 2023 💀💀✌

**Comment by BroJac5246** [comment, marked as answer]
It's not possible for us to do this. You would have to request it to the Scratch Team directly.

**Reply by CST1229** [reply]
And the ST has [rejected](https://scratch.mit.edu/discuss/topic/343602/) it.

**Reply by Alias1XP** [reply]
Oh im sorry. i didn't notice that until now. sorry if i bugged you
